examples/notebooks/hyperdrive.ipynb

```diff
@@ -137,122 +135,47 @@
       },
       "outputs": [],
       "source": [
-        "class RandomAgent(Agent):\n",
-        "    \"\"\"\n",
-        "    Agent that randomly opens or closes longs or shorts\n",
+        "class RandomAgent(random_agent.Policy):\n",
+        "    \"\"\"Agent that randomly opens or closes longs or shorts\n",
+        "\n",
+        "    Customized from the policy in that one can force the agent to only open longs or shorts\n",
         "    \"\"\"\n",
         "\n",
         "    def __init__(self, rng: Generator, trade_chance: float, wallet_address: int, budget: int = 10_000) -> None:\n",
         "        \"\"\"Add custom stuff then call basic policy init\"\"\"\n",
         "        self.trade_long = True  # default to allow easy overriding\n",
         "        self.trade_short = True  # default to allow easy overriding\n",
```

Comment:
why not add these to the random agent class? it already has some customizability in terms of `trade_chance` this would just add the ability for directional randomnes. that would save you from having to sub-class anything in this notebook, i think.

Reply:
doing so would eliminate the instructive benefit of showing how to sub-class an agent tho 🤔

Reply:
If we find that we are subclassing for the same reason a lot down the road we can add it in.